src/app/containers/MostWatched/index.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useEffect, useContext } from 'react';
+import 'isomorphic-fetch';
+import { arrayOf, shape, bool, string } from 'prop-types';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
+import useToggle from '#hooks/useToggle';
+import StoryPromo from '../StoryPromo';
+import Grid from '../../components/Grid';
+import CpsOnwardJourney from '../CpsOnwardJourney';
+import { getMostWatchedData, processMostWatched } from './utilities';
+
+const MostWatched = ({ initialData, hasHeader }) => {
+  const { dir, mostWatched, service } = useContext(ServiceContext);
+  const { variant, isAmp } = useContext(RequestContext);
+
+  const defaultMostWatchedConfig = {
+    header: 'Most Watched',
+    numberOfItems: 10,
+    hasMostWatched: false,
+  };
+
+  const { header, numberOfItems, hasMostWatched } =
+    mostWatched || defaultMostWatchedConfig;
+
+  const { enabled } = useToggle('mostWatched');
+
+  const isMostWatchedEnabled = enabled && hasMostWatched;
```

Comment:
We should be able to rely completely on the toggle value only (and remove the service config for `hasMostWatched`)

Reply:
We are relying on the toggles now but we need to update them first and maybe schedule it's removal for another PR